var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py

```diff
@@ -13,6 +13,7 @@ class PyMorphologyRepairWorkflow(PythonPackage):
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/nse/morphology-repair-workflow"
     git      = "ssh://bbpcode.epfl.ch/nse/morphology-repair-workflow"
     version('develop', branch='master')
+    version('2.0.2', tag='morphology-repair-workflow-v2.0.1')
```

Comment:
Tag label need the last character to be 2.

Reply:
Good catch! The `packages/*.yaml` are superseded by `environments/*.yaml`, and should normally not have any version pins any longer

Reply:
So we can skip `deploy/packages/bbp-packages.yaml` update even before you'll merge https://github.com/BlueBrain/spack/pull/1013 ?

Reply:
Ah, nice :) now I see where the old jupyter stuff is coming from :) `py-jupyter-notebook@4.2.3` I'll rename it with next commit.